src/Fonts.scss

```diff
@@ -0,0 +1,102 @@
+//h1 will be nasa font
+// open sans for header 2, 3
+//roboto for p
+
+@import url('https://fonts.googleapis.com/css?family=Open+Sans');
+
+.opensans {
+  font-family: "Open Sans", sans-serif;
+}
+
+@import url('http://fonts.cdnfonts.com/css/bambi');
+
+.bambi{
+    font-family:'Bambi Bold', sans-serif;
+}
+
+@import url('https://typodermicfonts.com/nasalization/');
+
+.nasafont{
+    font-family:''
+  
+}
+// @font-face {
+//   font-family: 'Nasa';
+//   src: local('Nasa'), url(fonts/Nasa.ttf) format('tff');
+// }
+//import 'Nasa.tff' from 
+
+
+
+h1{
+  font-family: 'Bambi';
+  // font-family: nasalization, sans-serif;
```

Comment:
I think we can delete this commented code

Reply:
Deleted :)